libs/sfm/bundler_common.cc

```diff
@@ -266,5 +266,50 @@ load_survey_from_file (std::string const& filename,
     in.close();
 }
 
+/* ---------------------- Feature undistortion -------------------- */
+
+namespace {
+    double distort_radius (double const r2, float const* params)
+    {
+        double coeff = 1.0 + r2 * params[0] + r2 * r2 * params[1];
+        return r2 * coeff * coeff;
+    }
+
+    double solve_undistorted_radius (double const r2, float const* params)
+    {
+        // Guess initial interval upper and lower bound
```

Comment:
Is there a formula to compute this close form, instead of bisecting it?

Reply:
added comment

Reply:
We would have to find and test all roots of a large polynomial, which is not practical.